Woodoku/src/config.yaml

```diff
@@ -0,0 +1,21 @@
+raw_shapes:
+  - [[0, 0]]
+  - [[0, 0], [0, 1]]
+  - [[0, 0], [1, 1]]
+  - [[0, 0], [0, 1], [1, 0]]
+  - [[0, 0], [0, 1], [0, 2]]
+  - [[0, 0], [1, 1], [2, 2]]
+  - [[0, 0], [0, 1], [1, 1], [1, 2]]
+  - [[0, 0], [0, 1], [0, 2], [1, 0]]
+  - [[0, 0], [0, 1], [0, 2], [1, 1]]
+  - [[0, 0], [0, 1], [0, 2], [1, 2]]
+  - [[1, 0], [1, 1], [0, 1], [0, 2]]
+  - [[0, 0], [0, 1], [0, 2], [0, 3]]
+  - [[0, 0], [1, 1], [2, 2], [3, 3]]
+  - [[0, 0], [0, 1], [1, 0], [1, 1]]
+  - [[0, 0], [0, 1], [0, 2], [1, 0], [2, 0]]
+  - [[0, 0], [0, 1], [0, 2], [1, 1], [2, 1]]
+  - [[0, 0], [0, 1], [0, 2], [1, 2], [2, 2]]
+  - [[1, 0], [0, 1], [1, 1], [1, 2], [2, 1]]
+  - [[1, 0], [0, 0], [0, 1], [0, 2], [1, 2]]
+  - [[0, 0], [0, 1], [0, 2], [0, 3], [0, 4]]
```

Comment:
I also saw the following shape in the game
[[0, 2], [1, 2], [2, 0], [2, 1], [2, 2]]
[[0, 1], [1, 1], [1, 0]]
[[0, 0], [1, 0], [2, 0], [2, 1], [2, 2]]
[[0, 0], [0, 2], [1, 0], [1, 1], [1, 2]]
[[0, 1] ,[1, 0], [1, 1], [1, 2], [2, 1]]
[[0, 1], [1, 1], [2,0], [2, 1], [2, 2]]

Reply:
> [[0, 2], [1, 2], [2, 0], [2, 1], [2, 2]]

This can be generated by rotating `[[0, 0], [0, 1], [0, 2], [1, 0], [2, 0]]`.

> [[0, 1], [1, 1], [1, 0]]

This can be generated by rotating `[[0, 0], [0, 1], [1, 0]]`.

> [[0, 0], [1, 0], [2, 0], [2, 1], [2, 2]]

This can be generated by rotating `[[0, 0], [0, 1], [0, 2], [1, 1], [2, 1]]`.

> [[0, 0], [0, 2], [1, 0], [1, 1], [1, 2]]

This can be generated by rotating `[[1, 0], [0, 0], [0, 1], [0, 2], [1, 2]]`.

> [[0, 1] ,[1, 0], [1, 1], [1, 2], [2, 1]]

This is the same as `[[1, 0], [0, 1], [1, 1], [1, 2], [2, 1]]`.

> [[0, 1], [1, 1], [2,0], [2, 1], [2, 2]]

This can be generated by rotating `[[0, 0], [0, 1], [0, 2], [1, 1], [2, 1]]`.